docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -157,6 +157,38 @@ The most commonly used CR settings are listed in this table:
 |`analyzer_container_requests_memory`
 |`4Gi`
 |Minimum amount of memory the pod needs to run
+
+|`ui_container_limits_cpu`
+|`500m`
+|Maximum number of CPUs the UI pod resource is allowed to use
+
+|`ui_container_limits_memory`
+|`800Mi`
+|Maximum amount of memory the UI pod resource is allowed to use. You can increase this limit if the pod displays `OOMKilled` errors.
+
+|`ui_container_requests_cpu`
+|`100m`
```

Comment:
@dymurray && @istein1 - please can i ask for a check on this value given in the Jira ticket as it does not look correct? Should these values not be an integer value? thanks